### PR TITLE
migration file created using ERD

### PR DESCRIPTION
**Pull request by kajiwara12**
Created the migration file seed file still needs to be worked on

### CODE REVIEW

File: .env.template
Comment (AGI-CEO) [inline .env.template:1]: why was .env.template deleted? Generally this should be included in the repo and built upon

**Comment by gnprwx** [comment]
Is this relevant anymore? The ERD changed according to https://github.com/mcsp-blue-ocean/quest/issues/20

File: .env.template
Comment (gnprwx) [reply .env.template:1]: This should stay, indeed.

**Review by gnprwx — CHANGES_REQUESTED** [review]
Retain .env.template and update the migration.sql file to reflect the most recent ERD generated.

**Pull request closed without merging**